compatibility/BUILD

```diff
@@ -79,9 +79,12 @@ config_setting(
     if versions.is_at_least(sdk_version, platform_version)
 ]
 
+last_pre_7456_trigger_version = "1.6.0-snapshot.20200930.5312.0.b9a1905d"
+
 [
     daml_trigger_dar(sdk_version)
     for sdk_version in sdk_versions
+    if not versions.is_at_most(last_pre_7456_trigger_version, sdk_version)
```

Comment:
This seems a bit too strong. We don’t want to kill those compat tests for newer versions completely. What we can remove is testing a version > last_pre_7456_trigger_version against a DAR built with an older version.

Reply:
What I'm trying to do here is avoid building the trigger dar with any SDK version that precedes this change. Is that not what I'm expressing here?

Reply:
duh, I somehow missed the negation. Seems sensible then.